mne/annotations.py

```diff
@@ -303,6 +303,44 @@ def read_annotations(fname):
     return annotations
 
 
+def read_brainstorm_annotations(fname, orig_time=None):
+    """Read annotations from a Brainstorm events_ file.
+
+    Parameters
+    ----------
+    fname : str
+        The filename
+    orig_time : meas_date
+        The 0 timestamp 
+
+    Returns
+    -------
+    annot : instance of Annotations | None
+        The annotations.
+    """
+    from scipy import io
```

Comment:
imports should be placed at the top of the file. 

I would also do `from scipy.io import loadmat` to avoid loading the whole `io`, and then use it directly like:
```diff
- annot_data = io.loadmat(fname)
+ annot_data = loadmat(fname)
```
if anyone argues that `io.loadmat` is more readable than `loadmat` and that the benefit of not loading the entire `io` module is lost in readability. I retract 'cos I agree that `io.loadmat` is more readable. And readability is always first.

Reply:
> to avoid loading the whole io

Since `loadmat` is a function within `io`, doing `from scipy.io import loadmat` and `from scipy import io` should have the same overhead, if that's what you're concerned with.

Either one looks fine to me. We use both in the codebase, but `from scipy.io import loadmat` is the more common one.